### ISSUE TITLE
Get Error When clicking on "Configurable Tier Prices"

### ISSUE THREAD

**Issue by sugarproject** [sprankhub/Spranks_ConfigurableTierPrices#17]
System Configuration :
- Windows 8.1 Pro
- PHP 5.5.20
- MySQL Server 5.6.22
- Magento 1.9.1

The installation has successfully done. Then I clear all files under \var\cache. Logout and Login again the admin. Get Error When clicking on "Configurable Tier Prices"

PHP Fatal error:  Class 'Spranks_ConfigurableTierPrices_Helper_Data' not found in E:\Website_Magento\BlankTee191\app\Mage.php on line 547

Best Regards,

Billson

**Comment by sprankhub** [comment]
Never heard of that issue. Do you have compilation enabled? This might cause such an issue. And do you use the latest version?

**Comment by sugarproject** [comment]
Hi Simon,

Thank you your comments. It works after I have compilation disabled.

Billson

**Comment by sugarproject** [comment]
I found another issue.(May be I configure wrongly)
Under a configurable Product (with Color & Size are selectable), I set that Product's Tier Price
All Website, 
All Groups,
QTY 50 and above
Price xxx

It works ONLY for single SKU ordered 50pcs or above. If the carts contains 40 pcs of Red small size t-shirt and 20 pcs of Yellow medium t-shirts. Both items are original price.

Billson

**Comment by sprankhub** [comment]
Okay before you think the extension does not work with the compilation: It should work, but you have to recompile after each extension installation (not only after installing this extension - this is valid for all extensions).

I cannot really help you without more information. Have a look at the [FAQs](https://github.com/sprankhub/Spranks_ConfigurableTierPrices#faq). If it still does not work, send me the following information privately (contact information under http://simonsprankel.com):

- admin access to your store including admin URL, user name, password
- ID and link of the product where the tier price is defined, but not applied across multiple variations

**Comment by sugarproject** [comment]
I have sent you the logon details. Thank you!

**Comment by sprankhub** [comment]
Easy one: You configured under System - Configuration - Spranks Extensions - Configurable Tier Prices - General Settings - Disable Extension for the Following Categories that it should be disabled for all categories. If you do not want to disable it, you have to select "-- Disable for no Category --". That is what I did now and it works as expected now.

Hope you leave a good review on Magento Connect :-)

**Comment by sugarproject** [comment]
Hi Simon,

It works perfectly. Sorry for misunderstanding your confirg. Thank you very
much.

Best Regards,

Billson
